source/our_tools/tractoflow.rst

```diff
@@ -3,4 +3,220 @@
+
+        .. code-block:: bash
+
+            for a in input_data_tractoflow/*; do cat $a/bval; done
+
+    #. Tractoflow is 100% reproducible if you re-use in the future the SAME code and SAME container image (singularity). Hence, copy them locally in your project to SAVE them in the future.
+
+        .. code-block:: bash
+
+            cd ~/projects/def-descotea/USER
+            mkdir Project_X
+            cd Project_X
+            cp ~/tractoflow_*.img .
+            cp -r ~/tractoflow-* .
+
+    #. Now, we are ready to run tractoflow. Most important questions you should ask yourself:
+
+        - What shells (if you have multi-shell data) will I use to compute DTI metrics? *Typically: b < 1200*
+        - What shells (if you have multi-shell data) will I use to compute fODFs & metrics? *Typically: b>700 and the b0.*
+        - Do I want to fix the fiber response function (FRF) or compute the mean FRF for my group? *Typically: ?*
+        - What seeding strategy do you want for PFT tracking? Interface seeding or WM seeding? *Depends on your project. Ex, for a connectomics study, interface seeding is better.*
+        - How many seeds do you want? (or how many streamlines you wish to have in the final tractogram?) *Typically: ~2,000,000 - 3,000,000 streamlines. To calculate the number of seeds per voxel, you can use an approximation. Ex: on a test subject, we found 260k voxels of the wm-gm interface, from which we will seed. 260,000*15 = 3.9M seeds will be launched. Hopefully, this should lead to 2.5M-3.5M streamlines in the final tractogram.*
+
+       If any of the above four questions are puzzling or do not make sense, go back to your notes, readings, and courses. You don’t understand what you are about to launch! Or see Max or someone in the SCIL for help and an important discussion.
+
+       For example, we could launch the following command. However, **don't run it now**, we will actually use a sbatch (see lower).
+
+        .. code-block:: bash
+
+            # This would run tractoflow with the following parameters:
+            #   - Dti_shells 0 and 1500, Fodf_shells 0 and 1500.
+            #   - Fix the FRF to (15,4,4) x 10^-4 mm2/s
+            #   - Interface seeding
+            #   - nbr_seeds 15.
+            nextflow -c ../beluga.conf run ../tractoflow-*/main.nf --root ../input_data_tractoflow \
+                -with-singularity ../tractoflow_*.img -resume -with-report report.html \
+                --dti_shells "0 1500" --fodf_shells "0 1500" --step 0.5 --nbr_seeds 15 \
+                --wm_seeding false --mean_frf false --set_frf true
+
+    #. Before launching your command for real, let's test it quickly using an interactive node. You ask access to an interactive node doing this:
+
+        .. code-block:: bash
+
+            cd ~/scratch/Project_X
+            mkdir output_tractoflow
+            cd output_tractoflow
+
+            salloc -c 32 --mem 32G --time 00:10:00 -A def-descotea
```

Comment:
Less cpus please. If you ask this number just to test it is really big. Ask 8 or 12 cpus with 16G of RAM is enough.